addons/etcd/scripts/start.sh

```diff
@@ -0,0 +1,47 @@
+echo "start etcd..."
+
+
+export conf=/etc/etcd/etcd.conf
+export tmpconf=/tmp/etcd/etcd.conf
+
+cp $conf $tmpconf
+
+# peer tls check
+initial_advertise_peer_urls=$(sed -n 's/^initial-advertise-peer-urls: //p' $tmpconf)
+peer_protocol=""
+if echo "$initial_advertise_peer_urls" | grep -q "https" ; then
+    peer_protocol="https"
+else
+    peer_protocol="http"
+fi
+
+# when a member joins, initial-cluster needs to be reset because configmap will not update automatically
+MY_PEER=${KB_POD_FQDN}${CLUSTER_DOMAIN}
```

Comment:
Have a look at the code, KB_POD_FQDN doesn't include cluster domain, this looks unreasonable...

Reply:
wait to fix, maybe i should make an issue? :D